api/resources/translations/messages-es.properties

```diff
@@ -356,19 +356,19 @@ analytics.target.type = Tipo
 analytics.target.type.count = Cantidad
 analytics.target.type.percent = Por ciento
 analytics.targets = Objetivos
-analytics.targets.registrations = Familias Inscritas
+analytics.targets.registrations = Familias inscritas
 analytics.targets.unique.id = 
-analytics.unconfigured = Ningún modulo analitico configurado
+analytics.unconfigured = Ningún módulo analítico configurado
 and = y
 and\ should\ be\ sent\ at = y debería ser enviado a
 android_app.data_usage.app.title = Esta aplicación
-android_app.data_usage.description = N.B. Estos valores son desde el dispositivo se inició.
+android_app.data_usage.description = N.B. Estos valores son desde el dispositivo que se inició.
 android_app.data_usage.rx = Recibido
 android_app.data_usage.system.title = En todo el sistema
 android_app.data_usage.title = Uso de datos de la aplicación
 android_app.data_usage.tx = Transmitido
-android_app.version.title = Versión de Android app
-app.name = 
+android_app.version.title = Versión de la aplicación Android
+app.name = Nombre
```

Comment:
```suggestion
app.name = Aplicación
```
That is this link next to logout: 
<img width="307" alt="Screenshot 2024-02-20 at 6 20 32 PM" src="https://github.com/medic/cht-core/assets/66472237/6976f89f-2eb0-49f6-a067-38755aea2bde">